boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -376,6 +382,18 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
             break;
         }
     }
+    /* Check whether there would be more entries to process. */
+    if (PcBiosMapCount >= MAX_BIOS_DESCRIPTORS && Regs.x.ebx != 0x00000000)
```

Comment:
What's this good for? Wouldn't we have already printed and asserted above?

Reply:
There are 2 arrays, so we want a check for each.
No: above/inside is PcMapCount/MaxMemoryMapSize.

Reply:
Ah yes, I see. This is a while/else.